plutus-metatheory/src/Untyped/RenamingSubstitution.lagda.md

```diff
@@ -54,19 +61,30 @@ lift-cong : ∀{X Y}
 lift-cong p nothing  = refl
 lift-cong p (just x) = cong just (p x)
 
+renList-cong : ∀{X Y}
+    {ρ ρ' : Ren X Y}
+  → (∀ x → ρ x ≡ ρ' x)
+  → (t : List (X ⊢))
+  → renList ρ t ≡ renList ρ' t
```

Comment:
Ah I see, we need the function so we can prove this lemma about it?

Reply:
Same case of needing to satisfy the Termination Checker.